content/kubermatic/v2.18/architecture/known_issues/_index.en.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Known Issues"
+date = 2022-07-20T12:22:15+02:00
+weight = 1
+
++++
+
+## Overview
+
+This page documents the list of known issues description and possible work arounds/solutions.
+
+### 1. OIDC user authentication issue
+
+**Problem** 
+
+[OIDC](https://docs.kubermatic.com/kubermatic/v2.20/tutorials_howtos/oidc_provider_configuration/share-_clusters_via_delegated_oidc_authentication/) user is denied to access the user cluster in the KKP with K8s version of 1.20 and below. Refer the github issue [Bug: OIDC authentication...](https://github.com/kubermatic/kubermatic/issues/9908) for detailed problem description. Example logs look like below,
```

Comment:
Do not use absolute links, but local refs (e.g. `{{< ref "./master_seed/">}}` - see for example https://raw.githubusercontent.com/kubermatic/docs/master/content/kubermatic/v2.20/architecture/monitoring_logging_alerting/_index.en.md